tests/unit/aofrw.tcl

```diff
@@ -179,26 +179,25 @@ start_server {tags {"aofrw external:skip"} overrides {aof-use-rdb-preamble no}}
     }
 
     test {BGREWRITEAOF is delayed if BGSAVE is in progress} {
-        r multi
+        r config set rdb-key-save-delay 1000000
         r bgsave
-        r bgrewriteaof
-        r info persistence
-        set res [r exec]
-        assert_match {*scheduled*} [lindex $res 1]
-        assert_match {*aof_rewrite_scheduled:1*} [lindex $res 2]
+        assert_match {*scheduled*} [r bgrewriteaof]
+        assert_match {*aof_rewrite_scheduled:1*} [r info persistence]
```

Comment:
maybe use this opportunity to change this to use `[s aof_rewrite_scheduled]` instead of pattern matching

Reply:
OK